### PR TITLE
MTDSA-18540: Feature switch for OAS documentation

### PR DESCRIPTION
**Pull request by abigail-houston-hmrc**

### CODE REVIEW

File: app/config/AppConfig.scala [hunk 1]
```diff
@@ -50,15 +52,26 @@ trait AppConfig {
   def apiGatewayContext: String
   def confidenceLevelConfig: ConfidenceLevelConfig
   def apiStatus(version: Version): String
+
   def featureSwitches: Configuration
+  def endpointsEnabled(version: String): Boolean
   def endpointsEnabled(version: Version): Boolean
```
Comment (basotia) [inline app/config/AppConfig.scala:58]: we use to have Version Trait, but now it is back to String. Does this needs to be fixed?

File: conf/application.conf [hunk 2]
```diff
@@ -117,9 +120,11 @@ metrics {
 feature-switch {
   r10-fields {
     enabled = true
+    released-in-production = false
   }
   r10-IFS {
     enabled = true
+    releasedInProduction = true
```
Comment (basotia) [inline conf/application.conf:127]: this also has different name

Comment (abigail-houston-hmrc) [reply conf/application.conf:127]: It's not being used, so I'll remove it.

File: app/config/AppConfig.scala (hunk 1 above)
Comment (abigail-houston-hmrc) [reply app/config/AppConfig.scala:58]: It's overloaded, the first is used exclusively by the templates

**Review by basotia — APPROVED** [review]

**Pull request merged**